mypy/dmypy_server.py

```diff
@@ -153,14 +179,84 @@ def __init__(self, options: Options,
 
     def serve(self) -> None:
         """Serve requests, synchronously (no thread or fork)."""
+        if sys.platform == 'win32':
+            self.serve_win()
+        else:
+            self.serve_nix()
+
+    def serve_win(self) -> None:
+        if sys.platform == 'win32':
```

Comment:
Does `assert sys.platform == 'win32'` work? That would avoid all this indenting.

Reply:
Does ```
if sys.platform != 'win32':
    return
``` work?